.github/workflows/test_onnx.yml

```
@@ -27,7 +27,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        pip install .[tests,onnxruntime,exporters-tf]
+        pip install .[tests,onnxruntime,exporters]
```

we could even have instead
```suggestion
        pip install .[tests,exporters]
```